js/index.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable linebreak-style */
+const list = document.querySelector('.book-list');

```

Comment:
- [ ] Please don't ignore linter errors like this. These errors can break your application. Please remove this and remove the linter errors using `npx eslint . --fix` command :blush: It worked on my side :+1:

Reply:
Okay. Thanks
Done